HW1/P3/P3.txt

```diff
@@ -1 +1 @@
-# Your discussion here
+(u'aerst', 11, [u'arets', u'aster', u'rates', u'reast', u'resat', u'stare', u'stear', u'strae', u'tares', u'tears', u'teras'])
```

Comment:
If you take order and print the first few entries, you might find that there are actually 3 entries with 11 anagrams :)